rewards/df-intro.md

```diff
@@ -80,6 +89,10 @@ Challenge DF rewards Data Scientists that produce prediction models that consist
 
 You can visit the follow tutorial learn how to [access Challenge DF](../user-guides/data-farming/how-to-challengedf.md).  
```

Comment:
This is a big comment:

**Move all content in `../user-guides/data-farming` to DF section.**

**Why.** Having a bunch of DF content that isn't part of DF section is super-confusing. It's already caught me several times.
- This PR already fixes it for DF FAQs. Great!
- So let's also fix it for DF user guide:)

**How.** There are several possible ways to move it. Given time constraints, I'm sure you'll find a pragmatic way to move it over without any real rewriting. Options:
- Have the whole "user guide" section as another section side-by-side with other DF sections. Retain the "user guide" sub-section structure within DF.
- Or, have each sub-section of "user guide" section as its own section in DF, side-by-side with other DF sections.
- (I defer to you Roberto for which is best)

Reply:
Done. Updated links and redirects too.
```
Have the whole "user guide" section as another section side-by-side with other DF sections. Retain the "user guide" sub-section structure within DF.
```